### PR TITLE
add Volta and Turing arch

### PR DESCRIPTION
**Pull request by yajiedesign**
## Description ##
add Volta and Turing arch
Volta sm70
Turing  sm75

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [X] Changes are complete (i.e. I finished coding on this PR)
- [X] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [X] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [X] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

**Comment by ankkhedia** [comment]
@yajiedesign Thanks for your contribution!
Request you to provide more description on the PR so that it is easier for people to undertsand the PR.

@mxnet-label-bot [pr-awaiting-review]

@lebeg @larroy @marcoabreu Could you please help with the review?

**Comment by chinakook** [comment]
LGTM

**Comment by ankkhedia** [comment]
@marcoabreu @sandeep-krishnamurthy Could you please take a look and merge this if it is good to go?

File: cmake/FirstClassLangCuda.cmake [hunk 1]
```diff
@@ -120,6 +120,16 @@ else()
   list(APPEND CUDA_COMMON_GPU_ARCHITECTURES "5.2+PTX")
 endif ()
 
+if (CUDA_TOOLSET VERSION_GREATER "9.0")
+  list(APPEND CUDA_KNOWN_GPU_ARCHITECTURES "Volta")
+  list(APPEND CUDA_COMMON_GPU_ARCHITECTURES "7.0")
+else()
```
Comment (larroy) [inline cmake/FirstClassLangCuda.cmake]: elseif()?

Comment (larroy) [reply cmake/FirstClassLangCuda.cmake]: This looks wrong.

**Comment by kalyc** [comment]
Related https://github.com/apache/incubator-mxnet/pull/13168/files

**Comment by kalyc** [comment]
@mxnet-label-bot update [pr-awaiting-merge]

**Comment by kalyc** [comment]
@szha @nswamy could you please take a look?

**Comment by kalyc** [comment]
@yajiedesign the CI build appears to be failing - please re-trigger

**Comment by yajiedesign** [comment]
@szha test ok

**Comment by larroy** [comment]
@lebeg

**Comment by kalyc** [comment]
@mxnet-label-bot update [pr-awaiting-merge, cmake]

**Pull request merged**